js/testpkg/prophet.test.ts

```diff
@@ -76,4 +77,12 @@ describe('Prophet', () => {
         prophet.addSeasonality('daily', seasonality);
     });
   });
+
+  describe('regressors', () => {
+    it('can be set', () => {
+      const reg: ProphetRegressor = {mode: "additive", priorScale: 0.5, standardize: "auto"};
+      const prophet = new Prophet({optimizer});
+      prophet.addRegressor('feature1', reg);
+    });
+  });
```

Comment:
Could you add another test case or two after making everything optional to make sure the API works as expected? Perhaps one with the second argument omitted entirely and one with just `mode` provided.